freqtrade/templates/FreqaiExampleStrategy.py

```diff
@@ -47,11 +45,6 @@ class FreqaiExampleStrategy(IStrategy):
     startup_candle_count: int = 40
     can_short = False
 
-    linear_roi_offset = DecimalParameter(
```

Comment:
I think we need to keep at least one hyperoptable parameter around to show users example usage.

Reply:
Great idea.